### ISSUE TITLE
[BUG] - [PF2E] - Adding item to trade request from compendium causes trade to error and not complete

### ISSUE THREAD

**Issue by mProjectsCode** [fantasycalendar/FoundryVTT-ItemPiles#295]
**Have you tried to reset Item Piles' system settings?**
No

**Describe the bug**
When adding an item to the trading window from a compendium and then accepting the trade request causes the trade request to never complete.

**To Reproduce**

1. Make and accept a trade request (Merchant token for GM, player chooses their character)
2. Player adds item (money in this case)
3. GM adds item from compendium (or Quick Insert)
4. both accept trade request
5. trade request never completes
6. see error in the console

**Expected behavior**
The trade completes normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/61602848/225404730-1f838255-576a-4e77-b9c3-3798faf17285.png)

![image](https://user-images.githubusercontent.com/61602848/225404784-f4df282f-3e9e-4670-b197-10f1321f4219.png)

```
Uncaught (in promise) TypeError: undefined. Cannot read properties of undefined (reading 'toObject')
[Detected 2 packages: item-piles, socketlib]
    at trade-api.js:429:62
    at Array.map (<anonymous>)
    at TradeAPI._executeTrade (trade-api.js:428:42)
    at Object.executeTrade (socket.js:132:58)
    at SocketlibSocket._handleRequest (socketlib.js:254:9)
    at SocketlibSocket._onSocketReceived (socketlib.js:212:9)
    at x.emit (index.mjs:136:20)
    at i.value (socket.js:278:20)
    at i.value (socket.js:265:18)
    at i.value (socket.js:235:22)
    at x.emit (index.mjs:136:20)
    at s.value (manager.js:200:14)
    at x.emit (index.mjs:136:20)
    at a.value (index.js:119:51)
    at s.value (manager.js:192:22)
    at x.emit (index.mjs:136:20)
    at a.value (socket.js:323:26)
    at x.emit (index.mjs:136:20)
    at i.value (transport.js:104:14)
    at i.value (transport.js:97:14)
    at ws.onmessage (websocket.js:79:40)
(anonymous) @ trade-api.js:429
_executeTrade @ trade-api.js:428
executeTrade @ socket.js:132
_handleRequest @ socketlib.js:254
_onSocketReceived @ socketlib.js:212
x.emit @ index.mjs:136
value @ socket.js:278
value @ socket.js:265
value @ socket.js:235
x.emit @ index.mjs:136
value @ manager.js:200
x.emit @ index.mjs:136
value @ index.js:119
value @ manager.js:192
x.emit @ index.mjs:136
value @ socket.js:323
x.emit @ index.mjs:136
value @ transport.js:104
value @ transport.js:97
ws.onmessage @ websocket.js:79
```

**Setup:**

- Foundry Version: 10 (build 291)
- System Version: pf2e 4.9.2
- Item Piles Version: 2.4.19
- Browser & version: Standalone App for GM, Firefox 110.0.1 for Player

**Active modules:**

- Item Piles
- libwrapper
- socketlib
- quick insert

**Additional context**

**Comment by Haxxer** [comment]
Fixed in 2.5.0

**Comment by mProjectsCode** [comment]
Thanks for the quick fix
